src/app/workflow/executions/executions-tab.component.html

```diff
@@ -47,30 +54,66 @@
           <ng-container matColumnDef="minimum">
             <th mat-header-cell *matHeaderCellDef>Minimum</th>
             <td mat-cell *matCellDef="let executionMetric">
-              <span *ngIf="executionMetric.minimum" matTooltip="{{ executionMetric.minimum + ' ' + executionMetric.unit }}"
-                >{{ executionMetric.minimum | number: (executionMetric.metric === 'Cost' ? '1.2-2' : '1.0-2') }}
-                {{ executionMetric.unit }}</span
-              >
+              <span *ngIf="executionMetric.minimum !== null">
+                <!-- checking for null because minimum could be zero which is falsy -->
+                <span
+                  *ngIf="executionMetric.metric === 'Run Time'"
+                  matTooltip="{{ executionMetric.minimum | secondsToHoursMinutesSeconds }}"
+                  >{{ executionMetric.minimum | secondsToHoursMinutesSeconds }}</span
+                >
+                <span
+                  *ngIf="executionMetric.metric !== 'Run Time'"
+                  matTooltip="{{
+                    (executionMetric.minimum | number: (executionMetric.metric === 'Cost' ? '1.2-2' : '1.0-2')) + ' ' + executionMetric.unit
+                  }}"
+                  >{{ executionMetric.minimum | number: (executionMetric.metric === 'Cost' ? '1.2-2' : '1.0-2') }}
```

Comment:
I understand the objections to functions in an Angular template, but IMHO, it'd be better to centralize this logic in a function rather than have it duplicated six times.  Yes, you could put it in a pipe, but that's pretty heavy for such a simple task...

Reply:
I've de-duplicated it into an `ng-template`